packages/eslint-config-cozy-app/package.json

```diff
@@ -31,5 +30,11 @@
     "eslint-plugin-vue": "^8.1.1",
     "prettier": "^2.5.0",
     "typescript": "^4.5.2"
```

Comment:
@acezard : in my opinion,
- typescript, prettier should be devDependencies
- I wonder if like [eslint config airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json), we want to set eslint-config-prettier, eslint-plugin-* as peerDependencies + devDependencies
- I don't know what to think about @babel/eslint-parser, @typescript-eslint/*